jpype/_jexception.py

```diff
@@ -29,10 +29,6 @@ class JException(_jpype._JException, internal=True):
     ``java.lang.Throwable``.
 
     """
-    def __new__(cls, *args, **kwargs):
-        self = _jpype._JException.__new__(cls, *args)
-        _jpype._JException.__init__(self, *args)
-        return self
```

Comment:
The old new method is now complete removed.   As we deprecated the last of the use of JException as factory this is just a required stub for catching exceptions.